game.h

```diff
@@ -0,0 +1,15 @@
+#ifndef GAME_H
+#define GAME_H
+#include "character.h"
+
+
+class Game {
+	private:		
+		void printAttack(Character attacker, Character defender) const;
+		void printStatus(Character ch01, Character ch02) const;
+		void printWhoWins(Character ch01, Character ch02) const;
+
+	public:		
+		void fight(Character& ch01, Character& ch02) const;

```

Comment:
Logikusabb lenne, ha a karaktereket a Game tárolná. Főleg így ezeknek a metódusoknak nem sok értelme, akkor már lehetnének statikusak. És akkor ott vagyunk, hogy ez nem egy osztály, csak egy namespace. De ha tárolná a karaktereket, akkor értelmet kapna, és nem lennének ezek a felesleges függvény paraméterek. Amik most nm is jók, mert érték szerint vannak átadva a kiírásoknak.